aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -371,11 +414,16 @@ public class AwsProperties implements Serializable {
   private final Set<Tag> s3DeleteTags;
   private int s3FileIoDeleteThreads;
   private boolean isS3DeleteEnabled;
+  private boolean s3WriteTableUuidTagEnabled;
+  private boolean s3WriteTableNameTagEnabled;
+  private boolean s3WriteNamespaceTagEnabled;
   private final Map<String, String> s3BucketToAccessPointMapping;
 
   private String glueCatalogId;
   private boolean glueCatalogSkipArchive;
   private boolean glueLakeFormationEnabled;
+  private String tableName;
+  private String namespace;
```

Comment:
similarly, namespaceName

Reply:
This is not needed anymore as we are directly leveraging existing logic.